base/system/diskpart/diskpart_msg.mc

```diff
@@ -7219,84 +7236,88 @@ Example:
-                dismount before changing the partition type. When changing
-                the partition type, DiskPart will attempt to lock and dismount
-                the file system on the volume. If this parameter is not specified,
-                and the call to lock the file system fails, (because some other
-                application has an open handle to the volume), the entire
-                operation will fail. When this parameter is specified, the
-                dismount is forced even if the call to lock the file system
-                fails. When a file system is dismounted, all opened handles to
-                the volume will become invalid.
-
-    NOERR       For scripting only. When an error is encountered, DiskPart
-                continues to process commands as if the error did not occur.
-                Without the NOERR parameter, an error causes DiskPart to exit
-                with an error code.
-
-    Intended for Original Equipment Manufacturer (OEM) use only.
-
-    A partition must be selected for this operation to succeed.
-
-    Caution:
-
-        Changing partition type fields with this parameter might cause your
-        computer to fail or be unable to start up. Unless you are an OEM or an
-        IT professional experienced with GPT disks, do not change partition
-        type fields on GPT disks using this parameter. Instead, always use the
-        CREATE PARTITION EFI command to create EFI System partitions, the
-        CREATE PARTITION MSR command to create Microsoft Reserved partitions,
-        and the CREATE PARTITION PRIMARY command without the ID parameter to
-        create primary partitions on GPT disks.
-
-    This command does not work on dynamic disks nor on Microsoft Reserved
-    partitions.
-
-Example:
+                În afară de limitările menționate, DiskPart nu verifică
+                valabilitatea tipului de partiție decât pentru a se asigura că
+                este un octet în formă hexazecimală sau un GUID.
+
+    OVERRIDE    Activează DiskPart pentru a forța sistemul de fișiere pe volum
+                să demonteze tipul de partiție înainte de a-l schimba. Când se
+                schimbă tipul de partiție, DiskPart va încerca să blocheze și
+                să demonteze sistemul de fișiere pe volum. Dacă acest parametru
+                nu este specificat și apelarea blocării sistemului de fișiere
+                eșuează, (deoarece alte câteva aplicații au un identificator
+                de gestiune deschis pentru volum), întreaga operațiune va eșua.
+                Când acest parametru este specificat, demontarea e forțată
+                chiar dacă apelarea blocării sistemului de fișiere eșuează.
+                Când un sistem de fișiere e demontat, toate identificatoarele
+                de gestiune deschise ale volumului vor deveni nevalide.
+
+    NOERR       Doar pentru automatizare. Când apare o eroare, programul
```

Comment:
"For scripting only" gets translated here as "doar pentru automatizare", but in other instances as "doar pentru scriere de fișiere script". Please take care of consistency.